### ISSUE TITLE
Add overview for time spent on different projects

### ISSUE THREAD

**Issue by Maddosaurus** [super-productivity/super-productivity#2034]
First of all, thank you so much for super-productivity!
I scanned through the issues and found similar issues to my idea, though they may be a bit different to the use case I have in mind.
Related: #614 and #366 

## Problem Statement
I would like to see, on a daily/weekly/monthly basis, how much time I've spent relatively on my projects.
E.g. of all tracked time, I have spent: 40% Project A, 30% Project B, 30% Project C

## :grey_question: Possible Solution
This might be a great addition to the "Worklog" tab of "Today".
I would imagine:
- a breakdown of projects next to / under "Time Spent Total" listing the total time spent on each project , 
- accompanied by a pie chart showing the relative distribution between said projects.

From a technical standpoint, my gut feeling tells me that this information could be already available, as it is shown in each projects' individual worklog currently.

## :heavy_plus_sign: Additional context
I'd be up to helping out with a PR / review on that feature, if you'd like, although I should mention that my Frontend skills a quite dusty at this point.

**Comment by johannesjo** [comment]
Thanks for the suggestion! Makes definitely sense to me to add something like this.

A PR dealing with this would be very welcome! Thanks for the offer! Please let me know, if you have any questions.

**Comment by Maddosaurus** [comment]
Ah, sorry, didn't see that the bot closed this as stale. This is still relevant :)

**Comment by Maddosaurus** [comment]
@johannesjo could you reopen the ticket as well? Seems like I'm either not seeing that option or I'm not allowed to do that.
Thank you very much :)

That said, are you planning to join Hacktoberfest with this repo? I'd definitely love to get a PR in next month for a first rendition of this functionality!

**Comment by Maddosaurus** [comment]
This is still relevant, and I'm still interested in building the overview. I haven't had the time just yet 😸

**Comment by Maddosaurus** [comment]
Yes, the issue is still relevant. I'm in the process of testing locally right now :)

**Comment by Livvi** [comment]
Hi @Maddosaurus, how is the progress on this? I would really love to see this functionality included in Super Productivity!

**Comment by Maddosaurus** [comment]
This is still relevant, although I have not made any progress unfortunately

**Comment by Alexander-Linkov** [comment]
Hi! This is very useful feature. I have several projects, to which I want include my efforts each day/week in some proportions. So it would be very handy if I can open Metric or Worklog of Today to see how many time (absolute or relevant) I plan for each project. 
And also, It would be very handy to see how many time I really spent on each project during week/month/year.

**Comment by sangorys** [comment]
Please, this is a killing feature.

For my case, I need to declare the number of hours spent by project by week. This is the only reason why I cannot completely switch from Toggl to Super Productivity

by the way thanks so much. This app is a killer app !!! You are so brilliant

**Comment by bitdivine** [comment]
Where would a good place be for this in the UI?

**Comment by bitdivine** [comment]
Right-clicking on _one_ project opens a dropdown with entries such as "Metrics".  So it seems natural to me to add right-clicking on "Projects" to show a similar drop-down with an entry for showing time breakdown by project.  Does that sound OK?

**Comment by Zoriot** [comment]
So i think that would either be a new section or opened from the Project Metrics. Maybe it's also possible to implement iwith a plugin but i guess not. 
@johannesjo what do you think ui wise?
